OSSMDOC-373: Update control plane references (#49723)
openshift/openshift-docs: modules/ossm-auto-route.adoc

>     - bookinfo.example.com

Reviewer: This file is missing a content type attribute. Please verify that all the files in this PR have content type attributes and add any that might be missing.

Author: @rolfedh I've updated the files that were missing a content type attribute so that they now have one.

----

> Then, the following OpenShift Routes are created automatically. You can check that the routes are created with the following command. In this example, `istio-system` is the name of the {SMProductShortName} control plane project.

Reviewer: ```suggestion
This `Gateway` resource creates the following OpenShift Routes. You can check that the routes are created by using the following command. In this example, `istio-system` is the name of the {SMProductShortName} control plane project.
```
Making this less of a step so that this module doesn't cross over into being a procedure module.

Author: done
